src/app/components/ProjectsView.tsx

```diff
@@ -0,0 +1,206 @@
+                    className="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"
+                    type="text"
+                    placeholder="Project name"
+                  />
+                </div>
+                <div className="mb-6">
+                  <label
+                    className="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+                    htmlFor="grid-state"
+                  >
+                    Type
+                  </label>
+                  <div className="relative">
+                    <select
+                      value={newProjectType}
+                      onChange={(e) => setNewProjectType(e.target.value)}
+                      className="block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+                    >
+                      <option
+                        disabled={
+                          !entitlements['project-types'].includes('Single')
+                        }
+                      >
+                        Single
+                      </option>
+                      <option
+                        disabled={
+                          !entitlements['project-types'].includes('Random')
+                        }
+                      >
+                        Random
+                      </option>
+                      <option
+                        disabled={
+                          !entitlements['project-types'].includes('Batch')
+                        }
+                      >
+                        Batch
+                      </option>
+                    </select>
+                    <div className="pointer-events-none absolute inset-y-0 right-0 flex items-center px-2 text-gray-700">
+                      <svg
+                        className="fill-current h-4 w-4"
+                        xmlns="http://www.w3.org/2000/svg"
+                        viewBox="0 0 20 20"
+                      >
+                        <path d="M9.293 12.95l.707.707L15.657 8l-1.414-1.414L10 10.828 5.757 6.586 4.343 8z" />
+                      </svg>
+                    </div>
+                  </div>
+                </div>
+              </div>
+
+              <div className="mt-4 flex justify-end gap-x-2">
+                <button
+                  type="button"
+                  className="rounded-md border border-transparent bg-gray-400 px-4 py-2 text-sm font-medium text-white hover:bg-opacity-90"
+                  onClick={() => setIsOpen(false)}
+                >
+                  Cancel
+                </button>
+                <button
+                  type="button"
+                  className="rounded-md border border-transparent bg-green-500  px-4 py-2 text-sm font-medium text-white hover:bg-opacity-90"
```

Comment:
Remove extra space after bg-green-500

Consider consolidating these button classes into a single class as the only diff between buttons here is the color. Maybe a basic button class for the entire app in globals.css?